src/url-loader/fs-url-loader.ts

```diff
@@ -29,13 +29,13 @@ export class FSUrlLoader implements UrlLoader {
   root: string;
 
   constructor(root?: string) {
-    this.root = root || '';
+    this.root = usePathSep(root || '');
```

Comment:
The root is just a string and it could come from a file that had path in a relative 'path/to/here' form.